cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/remote/RemotePrivilegesTestBase.java

```diff
@@ -145,6 +146,12 @@ public void testAuthorizationEnforcer() throws Exception {
     authorizationEnforcer.enforce(NS, ALICE, EnumSet.allOf(Action.class));
     authorizationEnforcer.enforce(APP, ALICE, Action.ADMIN);
     authorizationEnforcer.enforce(PROGRAM, ALICE, Action.EXECUTE);
+    try {
+      authorizationEnforcer.enforce(NS, BOB, Action.ADMIN);
+      Assert.fail();
+    } catch (UnauthorizedException e) {
```

Comment:
If your test expect s an exception, you can add it like this

@ Test(expected = Unauthorised Exception.class)

https://github.com/junit-team/junit4/wiki/Exception-testing